conventional_precommit_linter/helpers.py

```diff
@@ -0,0 +1,26 @@
+def _color_bold_green(text: str) -> str:
```

Comment:
if we consider multi-os support, here you may use `colorama` or `rich`.

Reply:
Thanks! Implemented with `colorama`.